data/saas/config/mixpanel_config.yml

```diff
@@ -0,0 +1,99 @@
+saas_config:
+  fides_key: mixpanel_connector_example
+  name: Mixpanel SaaS Config
+  description: A sample schema representing the Mixpanel connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: api_secret
+    - name: token
+    - name: gdpr_oauth_token
+    # TODO - OPTION wasn’t supported at the time it was written.
+    - name: domain
+    - name: email
```

Comment:
Remove email identity, same reasoning as the Gorgias connector https://github.com/ethyca/fidesops/pull/304/files#r830290394

Reply:
Done